tests/openvino/test_modeling.py

```diff
@@ -1932,13 +2002,50 @@ def test_compare_to_transformers(self, model_arch):
             self.assertIsInstance(getattr(ov_model, additional_part), MODEL_PARTS_CLS_MAPPING[additional_part])
         self.assertIsInstance(ov_model.config, PretrainedConfig)
         inputs = ov_model.preprocess_inputs(**preprocessors, text=prompt, image=self.IMAGE.resize((600, 600)))
-        # pytorch minicpmv and internvl are not designed to be used via forward
+        transformers_inputs = copy.deepcopy(inputs)
+        ov_model.to("AUTO")
+        self.assertEqual(ov_model._device, "AUTO")
+        self.assertEqual(ov_model.vision_embeddings._device, "AUTO")
+        self.assertTrue(ov_model.vision_embeddings.request is None)
+        self.assertEqual(ov_model.language_model._device, "AUTO")
+        self.assertTrue(ov_model.language_model.request is None)
+        self.assertTrue(ov_model.language_model.text_emb_request is None)
+        for additional_part in ov_model.additional_parts:
+            self.assertEqual(getattr(ov_model, additional_part)._device, "AUTO")
+            self.assertTrue(getattr(ov_model, additional_part).request is None)
```

Comment:
There are quite a few repeating checks like this added in the PR. What do you think about replacing them with for-loops? Possibly introducing some helper property for `OVModelForVisualCausalLM` like `.submodels` (or something like this) will be helpful. I think it also will be helpful inside some of the `OVModelForVisualCausalLM` methods, like `.half()` for example.

Reply:
added components for model wrappers and submodels for ov.Model objects, please take a look

Reply:
Looks good!